### PR TITLE
Fixes for using files v2

### PR DESCRIPTION
**Pull request by horheynm**
Need to address
* deprecated models
* non-existing files (sample-originals, https://github.com/neuralmagic/deepsparse/pull/1233)
* Api call and appropriate file download (blocked on https://github.com/neuralmagic/sparsezoo/pull/363)
* Model input/output change with respect to the stub

### CODE REVIEW

File: src/deepsparse/transformers/pipelines/embedding_extraction.py [hunk 1]
```diff
@@ -100,10 +100,7 @@ def to_list(cls) -> List[str]:
 @Pipeline.register(
     task="transformers_embedding_extraction",
     task_aliases=[],
-    default_model_path=(
-        "zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/"
-        "wikipedia_bookcorpus/pruned80_quant-none-vnni"
-    ),
+    default_model_path=("zoo:bert-large-wikipedia_bookcorpus-pruned90"),
```
Comment (dbogunowicz) [inline src/deepsparse/transformers/pipelines/embedding_extraction.py:103]: But that's a different model from the initial one?

File: tests/deepsparse/image_classification/test_pipelines.py [hunk 2]
```diff
@@ -34,8 +34,7 @@
     "zoo_stub,image_size,num_samples",
     [
         (
-            "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/"
-            "pruned85_quant-none-vnni",
+            "zoo:mobilenet_v1-1.0-imagenet-pruned.4block_quantized",
```
Comment (dbogunowicz) [inline tests/deepsparse/image_classification/test_pipelines.py:37]: In general, many models seem to be replaced not by their equivalents, but totally different models.

File: src/deepsparse/transformers/pipelines/embedding_extraction.py (hunk 1 above)
Comment (horheynm) [reply src/deepsparse/transformers/pipelines/embedding_extraction.py:103]: Original models dont exist anymore. 
Replaced by models that share the same task

File: tests/deepsparse/image_classification/test_pipelines.py (hunk 2 above)
Comment (horheynm) [reply tests/deepsparse/image_classification/test_pipelines.py:37]: see above

**Review by bfineran — APPROVED** [review]

**Review by dbogunowicz — APPROVED** [review]

**Pull request merged**